docs/rules/camelcase.md

```diff
@@ -68,6 +99,12 @@ Examples of **correct** code for this rule with the `{ "properties": "never" }`
 var obj = {
     my_pref: 1
 };
+
+const { no_camelcased = false } = bar;
```

Comment:
I don't think we need this or this change any more, since this behavior is the same as the cases listed above.